code/lightna/lightna-redis/App/Storage/Redis.php

```diff
@@ -38,26 +44,53 @@ protected function connect(): self
         return $this;
     }
 
-    public function set(string $key, mixed $value): void
+    public function set(string $key, mixed $value, array $tags = []): void
```

Comment:
if you rebase you will find 2 new functions batch/flush. probably they also should be adopted